recipes-core/glibc/glibc/windows-31j_support.patch

```diff
@@ -0,0 +1,24 @@
+From fe8ae616254da00628b45b22500e384fb8c503a0 Mon Sep 17 00:00:00 2001
+From: Charlie Johnston <charlie.johnston@ni.com>
+Date: Mon, 9 May 2022 11:03:55 -0500
+Subject: [PATCH] Adding ja_JP.WINDOWS-31J to built locales
+
```

Comment:
Could you add an Upstream Status trailer here, [per the styleguide](https://github.com/ni/nilrt/blob/nilrt/master/hardknott/docs/CONTRIBUTING.md#patch-files)? And also note the rationale for the .patch in the summary.